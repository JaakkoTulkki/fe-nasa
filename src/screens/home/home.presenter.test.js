import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'
import React from 'react'

import Home from './home.presenter'

configure({ adapter: new Adapter() })

test('Home renders', () => {
  const wrapper = shallow(<Home />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
