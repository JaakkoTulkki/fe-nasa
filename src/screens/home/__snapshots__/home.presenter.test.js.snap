// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders 1`] = `
<div>
  <SearchBox
    title=""
  />
  <ThumbNailGallery />
  <Pagination />
</div>
`;
